api/main.py

```diff
@@ -105,6 +106,7 @@ def list_of_installed_models(
 ):
     return models
 
+THREADS = cpu_count()
```

Comment:
We should only use the number of assigned cores, not the physical ones. 

This can be done with:

```
THREADS = len(psutil.Process().cpu_affinity())
```

Reply:
Just fixed that, thanks for the suggestion